ideascube/forms.py

```diff
@@ -69,11 +69,24 @@ def save(self, *args, **kwargs):
 class UserImportForm(forms.Form):
     source = forms.FileField(required=True)
 
+    def canonicalize_row(self, row):
+        result = {}
+        data_fields = User.get_data_fields()
+        data_fields = {str(f.verbose_name): f.name for f in data_fields}
```

Comment:
As far as I know, the export has been used to backup data or to load backuped data within ideascube. So far, BSF never received an export from the users from an ideasbox.
However, the aimed goal is to have insight of who is using the box (age, sex, educationnal level, etc.).